addons/udes_stock_packaging/models/stock_quant_package.py

```diff
@@ -122,6 +122,25 @@ def _get_all_products_quantities(self):
             res[quant.product_id] += quant.quantity
         return res
 
+    def check_package_contains_single_item(self, raise_error=True):
+        """
+        Check if package has one item only.
+        Raise Error if raise_error boolean optional parameter is True.
+        """
+        self.ensure_one()
+        product_qtys = self._get_product_quantities()
+        if len(product_qtys) != 1 or sum(product_qtys.values()) != 1:
```

Comment:
Do we need to be concerned with float qtys?

Reply:
This was already in Jayshree's story, i just wrapped into a method, changed the method name and in both places called the new method. 
Not sure if we will have qty with floats like 0.5 or 1.3 etc, i think in all cases we should use box, didnt want to possible change behaviour on smth that is already merged